### PR TITLE
EPMRPP-74457 || Implement "Do you have questions" form without actions

### PR DESCRIPTION
**Pull request by Vadim73i**

### CODE REVIEW

File: src/react-components/button/button.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+const Button = ({ text, onClick }) => (
+  <div
+    className="button"
+    onClick={onClick}
+  >
+    {text}
+  </div>
+);
+Button.propTypes = {
+  text: PropTypes.string,
```
Comment (AmsterGet) [inline src/react-components/button/button.jsx]: Can you pass it through the `children` prop?

File: src/react-components/questions-form/questionsForm.scss [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+          border-bottom: 1px solid $COLOR-TOPAZ;
+        }
+
+        &.error {
+          border-bottom: 1px solid $COLOR--error-red;
+        }
+      }
+
+      .error-message {
+        display: table-cell;
```
Comment (AmsterGet) [inline src/react-components/questions-form/questionsForm.scss]: What is the purpose of displaying it as `table-cell` outside of the table?

File: src/react-components/questions-form/questionsForm.jsx [hunk 3]
```diff
@@ -1,7 +1,114 @@
...
+  const onClick = () => {
+    console.log('questions form send click');
+  };
+
+  // todo add validation for all handlers
+  const onFirstNameChange = (event) => {
+    const currentFirstName = event.target.value;
+    setFirstName(currentFirstName);
+    setFirstNameError(!currentFirstName);
+  };
```
Comment (chivekrodis) [inline src/react-components/questions-form/questionsForm.jsx]: onChange funcs looks the same for all fields, does it possible use only one func and manage setValue and onError via params?
Also, maybe better compile state into one object

File: src/react-components/questions-form/questionsForm.jsx [hunk 4]
```diff
@@ -1,7 +1,114 @@
...
+    setCompanyNameError(!currentCompanyName);
+  };
+
+  return (
+    <div className="questions-form">
+      <div className="header">Do you have a question?</div>
+      <div className="text">
+        For more details please leave your e-mail and we will contact you within 3 business days.
+      </div>
+      <div className="form">
```
Comment (chivekrodis) [inline src/react-components/questions-form/questionsForm.jsx:48]: I'm not sure but please make sure this form shouldn't be salesforce form

**Review by AmsterGet — CHANGES_REQUESTED** [review]

File: src/react-components/questions-form/questionsForm.jsx (hunk 4 above)
Comment (AmsterGet) [reply src/react-components/questions-form/questionsForm.jsx:48]: For now it should be just empty form without action. Salesform fields will be migrated to react in the scope of future task.

**Review by AmsterGet — APPROVED** [review]

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**